source/crud/tests/write/replaceOne-collation.yml

```diff
@@ -17,6 +17,7 @@ tests:
             result:
                 matchedCount: 1
                 modifiedCount: 1
+                upsertedCount: 0
```

Comment:
Previously, no tests were asserting `upsertedCount`. It seems like a freebie, since the field is available for all server versions.